consensus/clique/api.go

```diff
@@ -117,3 +119,60 @@ func (api *API) Discard(address common.Address) {
 
 	delete(api.clique.proposals, address)
 }
+
+type CliqueStatus struct {
+	InturnPercent float64                `json:"inturnPercent"`
+	SigningStatus map[common.Address]int `json:"sealerActivity""`
+	NumBlocks     uint64                 `json:"numBlocks"`
+}
+
+// Status returns the status of the last N blocks,
+// - the number of active signers,
+// - the number of signers,
+// - the percentage of in-turn blocks
+func (api *API) Status() (*CliqueStatus, error) {
+	var (
+		numBlocks = uint64(64)
```

Comment:
Hard coding 64 seems overly arbitrary. We should probably convert it into an optional parameter, defaulting to 64 if unspecified perhaps. Though even then 64 blocks is 16 minutes, not sure if that's relevant enough.

Reply:
Perhaps it would be cleaner to make this a mandatory argument (and also drop the `numBlocks` from the result)?

Reply:
I agree that `64` is a bit arbitrary, the reason I didn't set an optional parameter is that it would be too easy to send in a very large number and basically DoS the server. If we want larger span of time, then we'd probably have to do the work during normal operation, and have this method just deliver the data we've accumulated over time.